assets/src/state/searchPageState.ts

```diff
@@ -76,28 +78,37 @@ export const submitSearch = (): SubmitSearchAction => ({
   type: "SUBMIT_SEARCH",
 })
 
-interface ClearSearchAction {
-  type: "CLEAR_SEARCH"
+interface NewSearchSessionAction {
+  type: "NEW_SEARCH_SESSION"
+  payload: SelectedEntity | null
 }
-export const clearSearch = (): ClearSearchAction => ({
-  type: "CLEAR_SEARCH",
+export const newSearchSession = (
+  newSelection?: SelectedEntity | null
+): NewSearchSessionAction => ({
+  type: "NEW_SEARCH_SESSION",
+  payload: newSelection || null,
 })
```

Comment:
Hmm... I see what you mean about how it might be confusing, but unfortunately I can't think of anything else. Maybe `newSearchContext`? But that's also overloaded due to React contexts.